cli/pkg/types/helmfile.go

```diff
@@ -2,113 +2,77 @@ package types
 
 import (
 	"errors"
-	"fmt"
 	"io"
 	"io/ioutil"
 	"os"
-	"tonic-install/pkg/helpers"
-	tonicio "tonic-install/pkg/io"
+	"path/filepath"
+	"strings"
+	h "tonic-install/pkg/helpers"
 
 	"gopkg.in/yaml.v2"
 )
 
-var ErrCouldNotWriteFile error = errors.New("could not write helm file")
-var writeValuesErrorMsg = "Looks like we had some trouble with that, do you want to try another path?"
-
-func WriteTonicChartValuesToFile(values map[string]interface{}, r io.Reader, w io.Writer) error {
-	choice, err := tonicio.YesNo(r, w, "Do you want to store a helm compatible values file for this deployment?", tonicio.NoDefault)
+func LoadOrCreateValuesFile(installValues TonicInstallValues, reader io.Reader, writer io.Writer) (*TonicChartValues, error) {
+	fileName, err := getFileName(installValues);
 	if err != nil {
-		return err
+		return nil, errors.New("Could not get chart values file name: " + err.Error());
```

Comment:
```suggestion
		return fmt.Errorf("Could not get chart values file name: %w", err)
```

For all of the `errors.New("..." + err.Error())` instances.

Reply:
no one ever lets me concatenate anymore!

Reply:
If `fmt.Errorf("... %w", err)` wasn't the "idiomatic" way of wrapping errors I don't think I'd mind since Go seems to have similar performance around `string + string` and `fmt.Sprintf("...", some,  more, string)`